plugins/iommu/fu-iommu-plugin.c

```diff
@@ -51,6 +51,13 @@ fu_iommu_plugin_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
 	fwupd_security_attr_set_result_success(attr, FWUPD_SECURITY_ATTR_RESULT_ENABLED);
 	fu_security_attrs_append(attrs, attr);
 
+	/* we might be able to fix this */
+	if (fu_kernel_is_cmdline_mutable()) {
```

Comment:
I don't think this is a thorough enough check.  Kate's old stuff was looking at specific combinations on the kernel command line to decide if it can be fixed or not.  If someone passes `amd_iommu=off` for example you can't turn the IOMMU on with `iommu=force` anymore.